README.md

```diff
@@ -1,9 +1,85 @@
-# Montreal Elixir Umbrella app
-
-For the official Montreal Elixir community website. http://www.montrealelixir.ca
+# Welcome to the Montreal Elixir Community website
 
 [![Build Status](https://travis-ci.org/montrealelixir/website.svg?branch=master)](https://travis-ci.org/montrealelixir/website)
 
-## Deploy to Heroku
+You've found the source code for our community website: http://www.montrealelixir.ca. The website
+is a single-page application aggregating several of our online properties:
+
+  * [Meetup.com](https://www.meetup.com/montrealelixir)
+  * [Twitter](https://twitter.com/montrealelixir)
+  * [YouTube](https://www.youtube.com/channel/UCftyx5k7K_0a3wIGRtE2YQw)
+  * [GitHub](https://github.com/montrealelixir)
+
+The application is built using:
+
+  * [Elixir](https://elixir-lang.org/): a dynamic, functional language designed for building scalable
+    and maintainable applications, and
+  * [Phoenix Framework](http://www.phoenixframework.org/): a productive web framework that does not
+    compromise speed and maintainability.
+
+The website has three main purposes:
+
+  * Promote the Elixir community in Montreal
+  * Inform the community members about events and new content
+  * Provide an opportunity to to learn about Elixir and Phoenix by contributing to an
+    open source project    
+
+## Getting started
+
+These instructions will get you a copy of the project up and running on your local machine for
+development and testing purposes. See Hosting and Deployment for notes on how to deploy the project
+on a live system.
+
+### Prerequisites
+
+We assume you have the following installed:
+
+  * Git
+  * Erlang
+  * Elixir
+  * Node
+  * PostgreSQL
+
```

Comment:
We'll leave this for now, we'll let folks work in whatever environment they want.